src/resources/action-map.js

```diff
@@ -53,6 +53,14 @@ module.exports = {
         }
       }
     }),
+    between: (l, { gte, lt }) => ({
```

Comment:
I've got it as an object for clarity on the range, but I'm open to:
```suggestion
    between: (l, gte, lt) => ({
```
Or even:
```suggestion
   between: (l, { 
     gte = null, 
     gt = null, 
     lte = null, 
     lt = null
   }) => {
     assert(![gte, gt, lte, lt].every(f => f === null));
     assert(gte === null !== gt === null);
     assert(lte === null !== lt === null);
```